modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/parse/NamedColorConverterFactory.java

```diff
@@ -0,0 +1,436 @@
+        colors.put("yellowgreen", new Color(154, 205, 50));
+        colors.put("antiquewhite", new Color(250, 235, 215));
+        colors.put("aqua", new Color(0, 255, 255));
+        colors.put("aquamarine", new Color(127, 255, 212));
+        colors.put("azure", new Color(240, 255, 255));
+        colors.put("beige", new Color(245, 245, 220));
+        colors.put("bisque", new Color(255, 228, 196));
+        colors.put("black", new Color(0, 0, 0));
+        colors.put("blanchedalmond", new Color(255, 235, 205));
+        colors.put("blue", new Color(0, 0, 255));
+        colors.put("blueviolet", new Color(138, 43, 226));
+        colors.put("brown", new Color(165, 42, 42));
+        colors.put("burlywood", new Color(222, 184, 135));
+        colors.put("cadetblue", new Color(95, 158, 160));
+        colors.put("chartreuse", new Color(127, 255, 0));
+        colors.put("chocolate", new Color(210, 105, 30));
+        colors.put("coral", new Color(255, 127, 80));
+        colors.put("cornflowerblue", new Color(100, 149, 237));
+        colors.put("cornsilk", new Color(255, 248, 220));
+        colors.put("crimson", new Color(220, 20, 60));
+        colors.put("cyan", new Color(0, 255, 255));
+        colors.put("darkblue", new Color(0, 0, 139));
+        colors.put("darkcyan", new Color(0, 139, 139));
+        colors.put("darkgoldenrod", new Color(184, 134, 11));
+        colors.put("darkgray", new Color(169, 169, 169));
+        colors.put("darkgreen", new Color(0, 100, 0));
+        colors.put("darkkhaki", new Color(189, 183, 107));
+        colors.put("darkmagenta", new Color(139, 0, 139));
+        colors.put("darkolivegreen", new Color(85, 107, 47));
+        colors.put("darkorange", new Color(255, 140, 0));
+        colors.put("darkorchid", new Color(153, 50, 204));
+        colors.put("darkred", new Color(139, 0, 0));
+        colors.put("darksalmon", new Color(233, 150, 122));
+        colors.put("darkseagreen", new Color(143, 188, 143));
+        colors.put("darkslateblue", new Color(72, 61, 139));
+        colors.put("darkslategray", new Color(47, 79, 79));
+        colors.put("darkturquoise", new Color(0, 206, 209));
+        colors.put("darkviolet", new Color(148, 0, 211));
+        colors.put("deeppink", new Color(255, 20, 147));
+        colors.put("deepskyblue", new Color(0, 191, 255));
+        colors.put("dimgray", new Color(105, 105, 105));
+        colors.put("dodgerblue", new Color(30, 144, 255));
+        colors.put("firebrick", new Color(178, 34, 34));
+        colors.put("floralwhite", new Color(255, 250, 240));
+        colors.put("forestgreen", new Color(34, 139, 34));
+        colors.put("fuchsia", new Color(255, 0, 255));
+        colors.put("gainsboro", new Color(220, 220, 220));
+        colors.put("ghostwhite", new Color(248, 248, 255));
+        colors.put("gold", new Color(255, 215, 0));
+        colors.put("goldenrod", new Color(218, 165, 32));
+        colors.put("gray", new Color(128, 128, 128));
```

Comment:
Sorry, a little confused -- so despite the comments, we are leaving this X11 "gray" set to the CSS value (128,128,128) rather than the darker X11 value (190, 190, 190)? I'm not an artist, just making sure I'm on the same page.